packages/edit-post/src/components/header/more-menu/index.js

```diff
@@ -13,24 +13,16 @@ import ToolsMoreMenuGroup from '../tools-more-menu-group';
 import OptionsMenuItem from '../options-menu-item';
 import WritingMenu from '../writing-menu';
 
-const ariaClosed = __( 'Show more tools & options' );
```

Comment:
I looked back to when this was introduced in #10900, and the related issue #7203.

@anevins12 Do you know if the opposing action states would be necessary to describe by a label, considering the button's role as a pop-up button? The button is [rendered with both `aria-haspopup="true"` and `aria-expanded={ isOpen }`](https://github.com/WordPress/gutenberg/blob/dda829527618a4a75a323b1db13b4a695a588b3e/packages/components/src/dropdown-menu/index.js#L67-L68), which seems like it would make these labels redundant. Or should a developer be required to provide both of these labels when rendering any DropdownMenu ?

Reply:
I think implementing `aria-haspopup` and `aria-expanded` would convey the "Show" bit of the label, but I do think the "more tools & options" bit is still valuable.

Reply:
I'd agree.
- `aria-haspopup` is a property (doesn't change dynamically) and informs users there's something to expand, should be used only for controls that require user activation 
- `aria-expanded` is a state (does change dynamically) and informs users on the "expanded state"
- "more tools & options" helps in differentiating this button from other "more" buttons but it's a bit longish and I'd consider to simplify. Also, controls names shouldn't change dynamically: they represent _what_ the control is about and should not represent the _state_. I'd reconsider it and simplify to "Tools & options" with no dynamic change. Better wording welcome!

Aside: applies also the block toolbar: it's a "More options/Hide options" which is slightly confusing and inconsistent.

Reply:
> I do think the "more tools & options" bit is still valuable.

Yes, to be clear, the label "More tools & options" remains ([source](https://github.com/WordPress/gutenberg/blob/dda829527618a4a75a323b1db13b4a695a588b3e/packages/edit-post/src/components/header/more-menu/index.js#L21)). As proposed, it will no longer switch between "Show more tools & options" and "Hide more tools & options".

>Also, controls names shouldn't change dynamically: they represent _what_ the control is about and should not represent the _state_.  I'd reconsider it and simplify to "Tools & options" with no dynamic change. Better wording welcome!

Thanks, that's exactly what I was hoping. The further simplification seems reasonable to me as well.